code/modules/prophunt/Exclude/items.dm

```diff
@@ -0,0 +1,229 @@
+/*
+	Items, Structures, Machines
+*/
+
+
+//
+// Items
+//
+
+/obj/item/holo

```

Comment:
more duplicate definitions?

Reply:
I was trying to exclude these bits of code because they were preventing the prop hunt console from compiling but it might be the cause of what's making some of the items stay after changing arenas. Maybe I should actually do my due diligence and try changing more then one value across all the relevant code.